site/content/blog/2022-03-01-whats-new-in-svelte-march-2022.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "What's new in Svelte: March 2022"
+description: "Shadow endpoints and update detection come to SvelteKit"
```

Comment:
I think we should remove mentions of Shadow endpoints - they're just called `endpoints` now: https://github.com/sveltejs/kit/issues/3532#issuecomment-1029277110

Reply:
And now they're "page endpoints" :smile:  https://kit.svelte.dev/docs/routing#endpoints-page-endpoints